Perl.gitignore

```diff
@@ -33,3 +33,7 @@ inc/
 /MANIFEST.bak
 /pm_to_blib
 /*.zip
+
+# Carton
+/local/
+cpanfile.snapshot
```

Comment:
All the examples in https://metacpan.org/pod/Carton seem to explicitly add and commit `cpanfile.snapshot`. That would suggest it should not be in `.gitignore` by default?

Reply:
I don't think `cpanfile.snapshot` should be ignored by default. @miyagawa could confirm.

Reply:
It depends. For a library i.e. CPAN module, the `cpanfile.snapshot` file should be excluded, like this PR does, since people would run it on different platforms with different perl versions.

For an "application" repository, i.e. your work app, `cpanfile.snapshot` is definitely intended to be committed.